scripts/covid19_india/cases_count_states_data/COVID19_cases_indian_states.tmcf

```diff
@@ -0,0 +1,30 @@
+Node: E:COVID19_cases_india->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19_Positive
```

Comment:
Earlier today when we used Christie's tool we realized that the StatVar DCIDs are slightly different (maybe changed recently, not sure): s/COVID_19/ConditionCOVID_19/

http://datacommons.org/browser/CumulativeCount_MedicalTest_ConditionCOVID_19_Positive

(here and elsewhere I think)